helm/another-ldap/templates/secrets.yaml

```
@@ -0,0 +1,18 @@
+{{- if .Values.secrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "common.names.fullname" . }}-config
```

secret name should not include `-config`

```suggestion
  name: {{ template "common.names.fullname" . }}
```